x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/parser/EqlParser.java

```diff
@@ -126,29 +127,6 @@ public void reportContextSensitivity(Parser recognizer, DFA dfa,
             this.ruleNames = ruleNames;
         }
 
-        @Override
-        public void exitDigitIdentifier(EqlBaseParser.DigitIdentifierContext context) {
-            Token token = context.DIGIT_IDENTIFIER().getSymbol();
-            throw new ParsingException(
-                    "identifiers must not start with a digit; please use double quotes",
-                    null,
-                    token.getLine(),
-                    token.getCharPositionInLine());
-        }
-
-        @Override
-        public void exitQuotedIdentifier(EqlBaseParser.QuotedIdentifierContext context) {
```

Comment:
Another useful callback to strip quotes from identifier `"where" WHERE condition` so the parser receives `where` instead of `"where"`.